unittests/contracts/eosio.mechanics/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We likely want to submodule this in instead of copying it.
Not sure if we need to copy this license file out with other license files.
Thoughts on best way to use this? @arhag @spoonincode @wanderingbort

Reply:
imo, since we don't distribute the file any further than the builddir (that is, we neither include it in a package nor as part of `make install`), we probably don't need to do anything with the license besides just ensure it's there next to the source file(s) it pertains to. As you've done.

As far as submodule vs not, I'm unsure. It does feel like a submodule might be more proper but then again it feels awfully heavy for something like this.

Reply:
> As far as submodule vs not, I'm unsure. It does feel like a submodule might be more proper but then again it feels awfully heavy for something like this.

@ClaytonCalabrese said he had to make minor modifications to it to work with our cmake setup. Probably the best course is to just copy it in as this PR has it.

Reply:
ah yeah, if we've modified it that's probably best